### ISSUE TITLE
[vuvuzela] Test script could use a test (Trac #1412)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1404]
The latest change in trigger-sim exposed a deficiency in the vuvuzela test script.  It showed that nothing downstream of the global trigger simulation is ever run, since by default the simulation filters untriggered frames, which should be rare for noise only events, and for the 10 events that script generates this was clearly the case.

The best route, I think, is to set FilterMode to False in the TriggerSim segment and add a module at the end of the chain and make a few simple checks.  They don't have to be extensive, but just a few simple sanity checks would be nice.

Migrated from https://code.icecube.wisc.edu/ticket/1412
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:53", 
    "description": "The latest change in trigger-sim exposed a deficiency in the vuvuzela test script.  It showed that nothing downstream of the global trigger simulation is ever run, since by default the simulation filters untriggered frames, which should be rare for noise only events, and for the 10 events that script generates this was clearly the case.\n\nThe best route, I think, is to set FilterMode to False in the TriggerSim segment and add a module at the end of the chain and make a few simple checks.  They don't have to be extensive, but just a few simple sanity checks would be nice.", 
    "reporter": "olivas", 
    "cc": "mjlarson@nbi.ku.dk", 
    "resolution": "fixed", 
    "_ts": "1550067233566919", 
    "component": "combo simulation", 
    "summary": "[vuvuzela] Test script could use a test", 
    "priority": "major", 
    "keywords": "", 
    "time": "2015-10-30T16:08:28", 
    "milestone": "", 
    "owner": "mjl5147", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160119T04:27:27: olivas changed milestone from "IceSim 5.0" to "IceSim 5.1"

**Comment by icecube-trac** [comment]
Trac update at 20170420T19:32:08: 
* mjl5147 changed milestone from "IceSim 5.1" to "May 2017"
* mjl5147 changed cc from "" to "mjlarson@nbi.ku.dk"

**Comment by icecube-trac** [comment]
Trac update at 20170423T12:56:39: 
* mjl5147 changed status from "new" to "closed"
* mjl5147 commented:

> In changeset 155219:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="155219"
> Updating the python test script. This now includes explicit tests on the number of frames coming out of CoincidenceAfterProcessing for noise triggers as well as the number of hits and triggers for each frame. Also fixed an issue where the seeding wasn't being set for testing, meaning that previous tests sometimes failed due only to the seeding. Finally closes #1404.
> ```


* mjl5147 changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:53: 
* olivas changed milestone from "May 2017" to ""
* olivas commented:

> Milestone May 2017 deleted
